backend/src/controllers/tasks.controller.ts

```diff
@@ -28,6 +29,32 @@ export class TasksController {
     return this.tasksService.insertTask(task);
   }
 
+  @Post('/new-comment/:taskId')
+  async insertComment(
+    @Param('taskId') taskId: number,
+    @Body() comment: Comment,
+  ): Promise<void> {
+    if (!comment.comment) {
+      console.log('Comment is missing.');
+      return;
+    }
+    if (comment.comment.length > 255) {
```

Comment:
Can we create a constant to be used here as the limit? So instead of this, we could get something like this:
```
const MAX_COMMENT_LENGTH = 255
if (comment.comment.length > MAX_COMMENT_LENGTH) {
```
If we define the `const` at the level of the class, if can be reused later when needed by other methods as well.
Also, in this way the code may be a little bit easier to be read by others who are reading the code for the first time.